### PR TITLE
Add pandas-summary recipe

### PR DESCRIPTION
**Pull request by proinsias**

### CODE REVIEW

**Comment by proinsias** [comment]
Hmm. I thought I had set this up correctly to skip the python3 builds. 
Any tips on how to fix the AppVeyor builds?

**Comment by proinsias** [comment]
Still to do:

- [ ] Include license_file when added to MANIFEST.in – see [issue](https://github.com/mouradmourafiq/pandas-summary/issues/5).

**Comment by jakirkham** [comment]
Thanks for putting this together and raising the license issue, @proinsias. A comment in the recipe regarding this will suffice for now.

File: recipes/pandas-summary/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    - pandas_summary
+
+  requires:
+    - pytest
+
+about:
+  home: https://github.com/mouradmourafiq/pandas-summary
+  license: MIT
+  license_family: MIT
+  # license_file: # Missing MANIFEST.in – https://github.com/mouradmourafiq/pandas-summary/issues/5
```
Comment (jakirkham) [inline recipes/pandas-summary/meta.yaml]: xref: https://github.com/mouradmourafiq/pandas-summary/issues/5

File: recipes/pandas-summary/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [win and py3k]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy
+    - pandas
```
Comment (jakirkham) [inline recipes/pandas-summary/meta.yaml]: Can we please drop `numpy` and `pandas` from the build dependencies? Unless of course we need them at build for some reason.

File: recipes/pandas-summary/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [win and py3k]
```
Comment (jakirkham) [inline recipes/pandas-summary/meta.yaml]: Why do we need this skip?

File: recipes/pandas-summary/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,48 @@
+{% set name = "pandas-summary" %}
+{% set version = "0.0.4" %}
```
Comment (jakirkham) [inline recipes/pandas-summary/meta.yaml]: I guess we can use `0.0.41`.

File: recipes/pandas-summary/meta.yaml (hunk 1 above)
Comment (jakirkham) [inline recipes/pandas-summary/meta.yaml]: Fixed in version `0.0.41`.

File: recipes/pandas-summary/meta.yaml (hunk 3 above)
Comment (proinsias) [inline recipes/pandas-summary/meta.yaml]: Due to a Windows/Python 3 issue with the ebooklib dependency, skip the Windows/Python 3 builds for now.

**Comment by patricksnape** [comment]
Re-ran - Travis is finished.

**Comment by proinsias** [comment]
@jakirkham – need anything else from me?

File: recipes/pandas-summary/meta.yaml (hunk 3 above)
Comment (jakirkham) [inline recipes/pandas-summary/meta.yaml]: Where is the `ebooklib` dependency? Seems to just be `numpy` and `pandas`.

File: recipes/pandas-summary/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+    - pandas_summary
+
+  requires:
+    - pytest
+
+about:
+  home: https://github.com/mouradmourafiq/pandas-summary
+  license: MIT
+  license_family: MIT
+  # license_file: # Missing MANIFEST.in - mouradmourafiq/pandas-summary#5
```
Comment (jakirkham) [inline recipes/pandas-summary/meta.yaml]: xref: https://github.com/mouradmourafiq/pandas-summary/issues/5

Comment (jakirkham) [inline recipes/pandas-summary/meta.yaml]: So this should now be present. Please add `license_file: LICENSE`.

**Comment by jakirkham** [comment]
Thanks @proinsias.

**Pull request merged**